Automattic/woocommerce-payments
# add link payment method for capture later option

Fixes #4821 

#### Changes proposed in this Pull Request
At this point Link payment method is not enabled when capture later flag is on. Since payments with Link can also be captured later, we should enable Link also when capture later flag is on.








#### Testing instructions





* Enable Link payment method
* Set 'Capture later' flag to on. (`WC -> Settings -> Payments-> Transactions->Issue an authorization on checkout, and capture later`) 
* create an order and pay it with Link
* no errors should occur.

-------------------


**Post merge**
